packages/react-instantsearch/src/widgets/SearchBox.js

```diff
@@ -8,6 +8,11 @@ import SearchBoxComponent from '../components/SearchBox.js';
  * @propType {string[]} [focusShortcuts=['s','/']] - List of keyboard shortcuts that focus the search box. Accepts key names and key codes.
  * @propType {boolean} [autoFocus=false] - Should the search box be focused on render?
  * @propType {boolean} [searchAsYouType=true] - Should we search on every change to the query? If you disable this option, new searches will only be triggered by clicking the search button or by pressing the enter key while the search box is focused.
+ * @propType {function} [onSubmit] - Intercept submit event sent from the SearchBox form container.
+ * @propType {function} [onReset] - Listen to `reset` event sent from the SearchBox form container.
```

Comment:
miss: [on*] - Will be forwarded to the actual used `input` allowing you to listen for keystrokes for examples.